tably.py

```diff
@@ -281,6 +346,28 @@ def arg_parser():
              'by passing either `-`, `/` or `0`. If `--no-header` is used, '
              'this argument is ignored.'
     )
+    parser.add_argument(
+        '-e', '--no-escape',
+        action='store_true',
+        help='If selected, do not escape special LaTeX characters '
+    )
+    parser.add_argument(
+        '-f', '--fragment',
+        action='store_true',
+        help='If selected, only output content inside tabular environment '
+             '(no preamble, table environment, etc.) '
+    )
+    parser.add_argument(
+        '-ff', '--fragment-skip-header',
+        action='store_true',
+        help='Equivalent to passing -k 1 -n -f '
+             '(suppress header when they are on the first row of .csv and pass -f) '
```

Comment:
Nitpicking:

Whitespace at the end of a line is needed when there is a line/text following it. When it is the final line, it should end with a period and no whitespace after it.